### PR TITLE
Initial ingest of BDNYC WISE Photometry

### PR DESCRIPTION
**Pull request by dr-rodriguez**
An initial ingest of WISE Photometry as stored in the BDNYC database. This includes data for 341 systems.

```python
t = db.query(db.Photometry.c.source).distinct().astropy()
t
<Table length=341>
         source         
         str24          
------------------------
 2MASS J00001354+2554180
 2MASS J00011217+1535355
 2MASS J00034227-2822410
 2MASS J00040288-6410358
 2MASS J00100009-2031122
 2MASS J00150206+2959323
 2MASS J00154476+3516026
 2MASS J00165953-4056541
 2MASS J00184613-6356122
 2MASS J00192626+4614078
                     ...
          TVLM 513-46546
         TVLM 831-161058
         TVLM 868-110639
                  TWA 26
                  TWA 28
ULAS J141623.94+134836.3
 USNO-B1.0 0757-00057276
                   VB 10
WISE J061135.13-041024.0
                Wolf 359
```

### CODE REVIEW

**Review by Will-Cooper — APPROVED** [review]
Looks good, haven't looked at every single .json but can't see any problems with those. Will be interesting to see non WISE photometry at some point!

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -4,10 +4,9 @@
 name: Database Integration Tests
 
 on:
-  push:
-    branches: [ main ]
   pull_request:
     branches: [ main ]
+  workflow_dispatch:  # manual execution
```
Comment (Will-Cooper) [inline .github/workflows/python-app.yml:10]: what do these changes mean?

Comment (Will-Cooper) [reply .github/workflows/python-app.yml:10]: nevermind I've seen #54 now!

**Comment by dr-rodriguez** [comment]
Right, it's too much to look at every single json which is why the tests are in place. I'm hoping to slowly populate things so we can have more focused pull requests. Maybe at some point we'll work to add in 2MASS or similar.

**Pull request merged**